ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -1301,3 +1301,8 @@ permission.description.can_view_tasks_tab = Display the Tasks tab in the app. If
 permission.description.can_view_unallocated_data_records = Allowed to see reports that have no assigned contact.
 permission.description.can_view_users = Allowed to get a list of all configured users.
 permission.description.can_write_wealth_quintiles = Allowed to update contacts with their wealth quintiles.
+upload.app.forms = Upload app forms
+installed.app.forms = Installed app forms
+form.icon = Form Icon
+form.id = Form Id
+form.title = Form Title
```

Comment:
These values are in what's called "title case" which means you capitalise every word. For consistency we don't use title case anywhere (with a couple of exceptions), so change these three values to "sentence case", eg: "Form icon" with a lower case "i".

Also, I know the original issue asked for this, but having "form" at the start of each value is redundant. They should just be "Icon", "Unique ID", and "Title". This means you can reuse the translation key "icon" for the first one.